lxd/db/cluster/open.go

```diff
@@ -211,6 +212,19 @@ INSERT INTO nodes(id, name, address, schema, api_extensions, arch, description)
 				return err
 			}
 
+			// Server admin group
+			stmt = `INSERT INTO auth_groups (name, description) VALUES ('admin', 'Server admin')`
```

Comment:
Do you think the description should contain shorthand? 

Also, since a group can have many members, should the name be pluralised?

e.g. `('admins', 'Server administrators')`

Reply:
@tomponline what are your thoughts on this?

Reply:
agree, with `('admins', 'Server administrators')`